.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,10 +7,9 @@ assignees: ''
 ---
 ## Describe the issue
 
-
 ## Explain how to reproduce the bug or problem
 
-```
+```text
```

Comment:
Does the "text" syntax specification make any difference?

if not, please take it out everywhere where added here.